sci_strategic_plugin/src/sci_strategic_plugin.cpp

```diff
@@ -394,4 +397,43 @@ double SCIStrategicPlugin::findSpeedLimit(const lanelet::ConstLanelet& llt) cons
   }
 }
 
+cav_msgs::MobilityOperation SCIStrategicPlugin::generateMobilityOperation()
+{
+    cav_msgs::MobilityOperation mo_;
+    mo_.header.timestamp = ros::Time::now().toNSec();
```

Comment:
timestamp should be in milisecond

Reply:
This matches the code from the port_drayage_plugin. should we discuss this?